distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -91,6 +91,34 @@ In the ${catalog-ui}, workspaces can be shared between users at different levels
 
 === Searching with ${catalog-ui}
 
+${catalog-ui} can create search queries of several types, include <<_text_search_details,text>>, <<_temporal_search_details,temporal>>, <<_spatial_search_details,spatial>>, <<_type_search_details,"type">>, and <<_advanced_search_details,advanced>> searches.
+
+==== Text Search Details
+
+${catalog-ui} provides a text search capability which searches across all textual data of the targeted data source.
+Text search capabilities include:
+
+Search for an exact word::
+`Text = apple` : Returns items containing the word "apple" but not "apples". Matching occurs on word boundaries.
+
+Search for the existence of items containing multiple words::
+`Text = apple orange` :  Returns items containing both "apple" and "orange" words. Words can occur anywhere in the item's metadata.
+
+Search using wildcards::
+`Text = foo*` :  Returns items containing words like "food", "fool", etc
```

Comment:
I believe there are edge cases with leading/trailing wildcards and multivalued attributes that we might want to document here.